tools/eslint-rules/index.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
this implementation is not what we will be going w/ as i believe i rewrote it in ESM and it was preferrable, but the concept of having these rules all being aggregated here prepared for import was the general idea that makes this model highly attractive (should be repoduced)

Reply:
if you look at this file today, the while license blob is gone:

Refs: https://github.com/ampproject/amphtml/blob/01b1240f8e88fee0d22260cf817838004996c3fa/build-system/eslint-rules/index.js#L1